grand/sim/efield2voltage.py

```diff
@@ -47,7 +49,9 @@ class Efield2Voltage:
       * Save output in ROOT format
     """
 
-    def __init__(self, f_input, f_output="", seed=None, padding_factor=1.0):
+    def __init__(self, f_input, f_output="", seed=None, padding_factor=1.0, du_type='GP300'):
+        # du_type='GP300' (using hfss simulations), 'GP300_nec' (using nec simulations), 'GP300_mat' (using matlab simulations), 'Horizon'
```

Comment:
Please move the comment just a bit up where the class documentation is (after Goals, there should be a Input parameters section, or something like that, explaining what each parameter does. @lwpiotr , do we have a template for documenting classes/functions?. Should we?)

Reply:
Fixed

Reply:
We don't have, especially that it depends on the automatic parser. I use sphinx with special script for now, but it sucks, so I need to check if perhaps Doxygen started to document dataclasses, then we could switch and fix the format.